src/NServiceBus.Testing.Fakes/obsoletes.cs

```diff
@@ -0,0 +1,25 @@
+#pragma warning disable 1591
+namespace NServiceBus.Testing
+{
+    using System;
+    using System.Threading.Tasks;
+    
+    public partial class TestableInvokeHandlerContext
+    {        
+        [ObsoleteEx(
+            Message = "HandleCurrentMessageLater has been deprecated.",
+            RemoveInVersion = "8",
+            TreatAsErrorFromVersion = "7")]
+        public Task HandleCurrentMessageLater()
+        {
+            throw new NotImplementedException();
+        }
+        
+        [ObsoleteEx(
+            Message = "HandleCurrentMessageLaterWasCalled has been deprecated.",
+            RemoveInVersion = "8",
+            TreatAsErrorFromVersion = "7")]
+        public bool HandleCurrentMessageLaterWasCalled => throw new NotImplementedException();
+    }
+}
+#pragma warning restore 1591
```

Comment:
I'm not really sure we want to use the `obsoletes.cs` approach like in the core repo here. The obsoletes will be shipped too and I'm not sure we gain anything by that? I'd rather have a "todo"-issue for the v8 release to make sure we remove all obsoletes.

Reply:
Why would that be better? Following the existing pattern makes us sure to remove them.